src/display/network_utils.js

```diff
@@ -27,9 +27,10 @@ function validateRangeRequestCapabilities({ getResponseHeader, isHttp,
   if (disableRange || !isHttp) {
     return returnValues;
   }
-  if (getResponseHeader('Accept-Ranges') !== 'bytes') {
-    return returnValues;
-  }
+  // Chrome doesn't return the Accept-Ranges header from cache
+  // if (getResponseHeader('Accept-Ranges') !== 'bytes') {
+  //   return returnValues;
+  // }
```

Comment:
This was probably due to the Access-Control-Expose-Headers not being set?
Anyway, we now pass the contentLength as a param to PDFJS.getDocument, so we know we support Ranges